src/containers/Tasks/RequestAttestation/RequestAttestation.tsx

```diff
@@ -29,7 +29,7 @@ export type RequestAttestationProps = {
   terms: sdk.IAttestedClaim[]
   quoteData?: sdk.IQuoteAttesterSigned
   receiverAddresses: Array<sdk.PublicIdentity['address']>
-  delegationId: sdk.IDelegationNode['id'] | null
+  delegationId: sdk.IDelegationNode['id'] | undefined
```

Comment:
In the SDK delegationId can be a string or null, are we sure we want this as possibly undefined?